micro-rdk/src/common/entry.rs

```diff
@@ -100,7 +100,10 @@ pub async fn serve_web_inner<S>(
     let (cfg_response, cfg_received_datetime) = app_client
         .get_app_config(Some(network.get_ip()))
         .await
-        .unwrap();
+        .unwrap_or_else(|e| {
```

Comment:
invalid config response gets handled further down when matching on RobotRepresentation, using an empty robot.

still verifying the behavior.

I like that this gets rid of an unwrap, but there is another option to handle this within get_app_config itself, returning a default ConfigResponse and logging the error.